assets/sass/okta-theme.scss

```diff
@@ -133,6 +133,30 @@
     /* set the helper's color value to the same color as the main-container's background */
       color: $primary-bg-color;
     }
+    // adjust password toggle on login page to accomodate for tooltip
+    .o-form-input-name-password {
+      .password-toggle {
+        .eyeicon {
+          right: 45px;
```

Comment:
is this not taken care by the PasswordBox itself? Like when you add `params: { showPasswordToggle: true }` to the password input, don't we already have the styles there for the icon position?
same question for the divider

Reply:
We need this change here, since the widget uses styles that are overwritten in `assets/common/shared` scss files.